.gitignore

```diff
@@ -5,7 +5,7 @@ __pycache__/
 .ipynb_checkpoints
 
 # python cache
-.py
+.pyc
 
 # optuna
-state_dict.pt
+.pt
```

Comment:
this is in your gitignore, but I see a `.pt` file added here. Is the file added by mistake?